manifests/charts/kube-bench/templates/cronjob.yaml

```diff
@@ -59,7 +59,7 @@ spec:
               image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
               imagePullPolicy: {{ .Values.image.pullPolicy }}
               command: [ "/bin/sh" ]
-              args: [ "-c", "kube-bench --json | curl -s -H \"Content-Type: application/json\" -X POST --data-binary @- {{ .Values.reportsurl }}" ]
+              args: [ "-c", "apk update && apk add curl && kube-bench --json | curl -s -H \"Content-Type: application/json\" -X POST --data-binary @- {{ .Values.reportsurl }}" ]
```

Comment:
@charisprose  I believe we have to change like 20+ other files in this same directory.

Reply:
There are a bunch of other job/cron job specs that will need to be updated in the same way. If you just look at the other files in this folder it will become clear.

Reply:
Yup, I mentioned this in the chat. Will update the rest on Monday.